### ISSUE TITLE
Very Poor Usability

### ISSUE THREAD

**Issue by sendpi** [melonproject/portal#360]
Slow interface
Difficult to set ether gas limits 
Difficult to add trades - doesn't even display enough decimal places
Throws erros on trades
Basically, I couldn't get a fund built

**Comment by schmidsi** [comment]
Thanks for your feedback. Be aware that this is still an early early bleeding edge version with focus on functionality. There was little to no effort made in terms of user experience/usability. 

That said, we really value your feedback. Let me go through your points:
- Slow interface: Communicating with the blockchain is slow indeed. We are working on caching optimisations to speed up at least the read-stuff. But caching comes with a cost: You need to trust the caching-instance which could front-run you. So if the choice is speed or accuracy, which one would you prefer?
- Difficult to set ether gas limits: Why would you set ether gas limits in the first place? Our goal is to hide this problem from the user. Furthermore, setting gas limits is done in MetaMask which is beyond our control.
- Difficult to add trades: Ether for example has 18 decimal places, they vary for each token. Would you really like to go that much into detail? [Kraken for example has reduced the precision lately](https://blog.kraken.com/post/1261/announcement-reducing-price-precision-on-all-trading-pairs/).
- Throws errors on trades: We are aware of that and working on it.
- Basically, I couldn't get a fund built: What stopped you from building a fund?
